### PR TITLE
Added new NoSQL resource

### PR DESCRIPTION
**Pull request by rgreen324**
## What does this PR do?
Add resource(s)

### Description
This book aims to provide an approachable introduction to Redis concepts by outlining many of the key-value store’s commands
### Why is this valuable (or not)?
Recent book about managing Redis database
### How do we know it's really free?
This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
### For book lists, is it a book? For course lists, is it a course? etc.
Book
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1512,6 +1512,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [CouchDB: The Definitive Guide](http://guide.couchdb.org)
 * [Extracting Data from NoSQL Databases: A Step towards Interactive Visual Analysis of NoSQL Data](http://publications.lib.chalmers.se/records/fulltext/155048.pdf) - Petter Nasholm (PDF)
 * [Graph Databases](http://graphdatabases.com)
+* [How To Manage a Redis Database](https://www.digitalocean.com/community/books/how-to-manage-a-redis-database-ebook) - Mark Drake (PDF, Epub)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:1515]:
```suggestion
* [How To Manage a Redis Database](https://www.digitalocean.com/community/books/how-to-manage-a-redis-database-ebook) - Mark Drake (PDF, EPUB)
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1469,6 +1469,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 ### MongoDB
 
+* [How To Manage a Redis Database](https://www.digitalocean.com/community/books/how-to-manage-a-redis-database-ebook) - Mark Drake (PDF, EPUB)
```
Comment (davorpa) [inline books/free-programming-books-langs.md:1472]: It was well in NoSQL. In other listings with several resources there are a category apart.

Could you restore it into this category?

**Review by davorpa — CHANGES_REQUESTED** [review]

**Comment by rgreen324** [comment]
I moved it back to NoSQL. Thanks

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**